packages/web/src/pages/upload-page/fields/CollectibleGatedDescription.tsx

```diff
@@ -0,0 +1,64 @@
+import { useCallback } from 'react'
```

Comment:
since this isnt a field, would it make sense to have the collectiblegatedfield be a folder to encapsulate the extra logic?